### PR TITLE
Closes #2231 - Add information stream to about_Redirection, clean up …

### LINKED ISSUE #2231: Document the information output stream introduced in PSv5 (number 6)
Output stream number `6`, the information stream, has existed since PSv5, yet it isn't documented in [about_Redirection](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md)

By contrast, [Write-Information](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Utility/Write-Information.md) has known about it for a while.

Version(s) of document impacted
------------------------------
- [x] Impacts 6.1 document
- [x] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------

- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by bobbytreed**
…content, and synchronize

Version(s) of document impacted
------------------------------
- [X] Impacts 6.1 document
- [X] Impacts 6.0 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-3.0&branch=pr-en-us-2622) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-4.0&branch=pr-en-us-2622) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-5.0&branch=pr-en-us-2622) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-5.1&branch=pr-en-us-2622) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-6&branch=pr-en-us-2622) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 115**: **[Warning]** Invalid file link:(~/3.0/microsoft.powershell.utility/Write-Information.md).

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C27%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806271458463660-2622%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tzB5memOxrGAljTzooiZxOD7zpaH8AsB8aoDqAIJgHI%3D&st=2018-06-27T15%3A05%3A11Z&se=2018-07-28T15%3A10%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md [hunk 1]
```diff
@@ -21,130 +21,120 @@ You can use the following methods to redirect output:
...
+|Stream # |Description  |Introduced in |
+|---------|---------|---------|
+|1|**Success** Stream|PowerShell 2.0|
+|2|**Error** Stream|PowerShell 2.0|
+|3|**Warning** Stream|PowerShell 3.0|
+|4|**Verbose** Stream|PowerShell 3.0|
+|5|**Debug** Stream|PowerShell 3.0|
+|*|All Streams|PowerShell 3.0|
+
+The PowerShell redirection operators are as follows, where `n` represents
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md]: Combine this into one paragraph.

The PowerShell redirection operators are as follows, where `n` represents 
the stream number. The **Success** stream ( `1` ) is the default if no stream is specified.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md [hunk 2]
```diff
@@ -21,130 +21,120 @@ You can use the following methods to redirect output:
...
+### Powershell redirection operators
 
-The redirection operators enable you to send particular types of output to
-files and to the success output stream.
+The redirection operators enable you to send streams of data to a file or the
+**Success** output stream.
 
-The PowerShell redirection operators use the following characters to represent
-each output type:
+The PowerShell redirection operators use the following numbers to represent
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md]: Duplicate of paragraph after table.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md [hunk 3]
```diff
@@ -21,130 +21,120 @@ You can use the following methods to redirect output:
...
 
-```
--   All output
-1   Success output
-2   Errors
-3   Warning messages
-4   Verbose output
-5   Debug messages
-```
+> [!NOTE]
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md]: Move this note after the table.

Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md]: Completed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md (hunk 2 above)
Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md]: Couldn't find this

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-3.0&branch=pr-en-us-2622) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-4.0&branch=pr-en-us-2622) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-5.0&branch=pr-en-us-2622) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-5.1&branch=pr-en-us-2622) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-6&branch=pr-en-us-2622) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 113**: **[Warning]** Invalid file link:(~/3.0/microsoft.powershell.utility/Write-Information.md).

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C27%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806271625413000-2622%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=q%2Fa5A4%2FaCKfnpp4hYrVPzrn3xbITA2eb9iEXk1oU4jI%3D&st=2018-06-27T16%3A31%3A39Z&se=2018-07-28T16%3A36%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-3.0&branch=pr-en-us-2622) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-4.0&branch=pr-en-us-2622) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-5.0&branch=pr-en-us-2622) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-5.1&branch=pr-en-us-2622) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-6&branch=pr-en-us-2622) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/4.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.0/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/5.1/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Redirection.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2231/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C27%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806271629188498-2622%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=U6ACxncpCj%2FhBQXSMrLcwNQBaqNAbFIrGUrr3jAqLpQ%3D&st=2018-06-27T16%3A43%3A14Z&se=2018-07-28T16%3A48%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**